QRCoder/Base64QRCode.cs

```diff
@@ -43,25 +33,81 @@ public string GetGraphic(int pixelsPerModule, string darkColorHtmlHex, string li
 
         public string GetGraphic(int pixelsPerModule, Color darkColor, Color lightColor, bool drawQuietZones = true, ImageType imgType = ImageType.Png)
         {
-            var base64 = string.Empty;
-            using (Bitmap bmp = qr.GetGraphic(pixelsPerModule, darkColor, lightColor, drawQuietZones))
+            if (imgType == ImageType.Png)
             {
-                base64 = BitmapToBase64(bmp, imgType);
+                var pngCoder = new PngByteQRCode(QrCodeData);
+
+                byte[] pngData;
+                if (darkColor == Color.Black && lightColor == Color.White)
+                {
+                    pngData = pngCoder.GetGraphic(pixelsPerModule, drawQuietZones);
+                }
+                else
+                {
+                    byte[] darkColorBytes;
+                    byte[] lightColorBytes;
+                    if (darkColor.A != 255 || lightColor.A != 255)
+                    {
+                        darkColorBytes = new byte[] { darkColor.R, darkColor.G, darkColor.B, darkColor.A };
+                        lightColorBytes = new byte[] { lightColor.R, lightColor.G, lightColor.B, lightColor.A };
+                    }
+                    else
+                    {
+                        darkColorBytes = new byte[] { darkColor.R, darkColor.G, darkColor.B };
+                        lightColorBytes = new byte[] { lightColor.R, lightColor.G, lightColor.B };
+                    }
+                    pngData = pngCoder.GetGraphic(pixelsPerModule, darkColorBytes, lightColorBytes, drawQuietZones);
+                }
+
+                return Convert.ToBase64String(pngData, Base64FormattingOptions.None);
             }
-            return base64;
+
+#if NETFRAMEWORK || NETSTANDARD2_0 || NET5_0 || NET6_0_WINDOWS
+            if (
+#if NET6_0_OR_GREATER
+                RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
+#else
+                true
+#endif
+                )
+            {
+                var qr = new QRCode(QrCodeData);
```

Comment:
![image](https://github.com/codebude/QRCoder/assets/2236234/1c7c0415-2a25-4fd0-ab73-67e16916840c)

Is there a way to suppress this warnings? (As we are aware of the situation.)

Reply:
I'll review.

Reply:
Maybe we can add `#pragma warning disable CA1416` here?